dbhydra/dbhydra_core.py

```diff
@@ -329,6 +350,12 @@ def close_connection(self):
     def initialize_migrator(self, other_db=None):
         self.migrator = Migrator(self, other_db)
 
+    def _convert_column_type_dict_from_python(self, column_type_dict):
+        typing_python_mapping = {k: self.typing_to_python_mapping.get(v, v) for k, v in column_type_dict.items()}
+        return {k: self.python_database_type_mapping[v] for k, v in typing_python_mapping.items()}
```

Comment:
Hard to read and grasp what the method really does. A docstring with an example would be beneficial here I think.